references/embedding/train.py

```diff
@@ -0,0 +1,149 @@
+import os
+
+import torch
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+from torch.utils.data.dataset import Subset
+from torch.nn.modules.distance import PairwiseDistance
```

Comment:
This doesn't seem to be used

Reply:
Forgot to remove these before the PR. Sorry.

Removed.